src/main/java/com/jcabi/github/RtAssignees.java

```diff
@@ -76,7 +76,7 @@
         this.entry = req;
         final Coordinates coords = repo.coordinates();
         this.request = this.entry.uri()
-            .path("/repos")
+            .path(repo.github().rootRepoPath())
```

Comment:
@kkashi01 This is another problem with adding the ``rootRepoPath()`` on the Github interface. It also becomes a *de facto* global utility method which is used throughout the library.